404.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset=utf-8>
+<style>
+body {
+  margin: 3em;
+}
+p {
+  font-family: Helvetica, Arial, sans-serif;
+}
+</style>
+</head>
+<body>
+  <p>Looking for the GOV.UK Prototype Kit training website?</p>
```

Comment:
I don't think a page _needs_ a heading to satisfy WCAG, but Axe and SiteImprove both complain about it, so worth considering.

Axe also complaining about the page not having a `main` landmark.

Reply:
If we have a heading it should be the same as the title, right?

Reply:
ah, title and H1 don't have to be the same.

For the main site, they are mostly the same; the title has the prefix: - GOV.UK Prototype Kit (where possible)